privacy_base/views/data_protection_menu_view.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<!-- Copyright 2018 Eficent Business and IT Consulting Services S.L.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<openerp>
+<data>
+
+    <record id="action_data_protection_partner_form" model="ir.actions.act_window">
+        <field name="name">Partners</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">res.partner</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[]</field>
+        <field name="filter" eval="True"/>
+    </record>
+
+    <menuitem id="parent_menu_data_protection"
+              name="Data Protection"
+              groups="group_data_protection_user"
+    />
+
+    <menuitem id="menu_data_protection_master_data"
+              name="Master Data"
+              parent="parent_menu_data_protection"
+              groups="group_data_protection_user"
+              sequence="2"
+    />
+    <menuitem id="menu_data_protection_partner"
+              name="Partners"
+              parent="menu_data_protection_master_data"
+              action="action_data_protection_partner_form"
```

Comment:
Why putting partners here? There's the contacts addon that gives a good high-level menu for them. I don't think we need this. :thinking:

Reply:
This was made by Eficent, and I agree with it: it is the entry point to search for Partners, which are central to Data Protection.